domain/src/test/java/org/oppia/android/domain/platformparameter/PlatformParameterModuleTest.kt

```diff
@@ -0,0 +1,87 @@
+package org.oppia.android.domain.platformparameter
+
+import android.app.Application
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.testing.platformparameter.FakePlatformParameterModule
+import org.oppia.android.testing.platformparameter.FakePlatformParameterSingleton
+import org.oppia.android.testing.platformparameter.TEST_PARAM_DEFAULT_VALUE
+import org.oppia.android.testing.platformparameter.TEST_PARAM_NAME
+import org.oppia.android.testing.platformparameter.TEST_PARAM_VALUE
+import org.oppia.android.testing.platformparameter.TestParam
+import org.oppia.android.util.platformparameter.PlatformParameter
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+import org.oppia.android.app.model.PlatformParameter as ParameterValue
+
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(application = PlatformParameterModuleTest.TestApplication::class)
+class PlatformParameterModuleTest {
+
+  @Inject
+  @TestParam
+  lateinit var testPlatformParameter: PlatformParameter<String>
+
+  @Inject
+  lateinit var platformParameterSingleton: PlatformParameterSingleton
+
+  private val mockPlatformParameterMap by lazy {
+    val testPlatformParameter = ParameterValue.getDefaultInstance().toBuilder()
```

Comment:
I think this can be replaced with `ParameterValue.newBuilder().setString(...).build()`

Reply:
Done